src/amo/components/SearchTools/index.js

```diff
@@ -10,20 +10,17 @@ import type { SearchFilters } from 'amo/components/AutoSearchInput';
 
 type Props = {|
   filters: SearchFilters,
-  pathname: string,
 |};
 
 export class SearchToolsBase extends React.Component<Props> {
   render() {
-    const { filters, pathname, ...otherProps } = this.props;
+    const { filters } = this.props;
 
     return (
       <Search
-        {...otherProps}
```

Comment:
What was it relying on this spread for?

Reply:
I don't know but I don't see any useful undocumented props that we could pass to the `Search` component...